test/specs/views/Card/CardContent-test.js

```diff
@@ -24,5 +24,6 @@ describe('CardContent', () => {
     mapValueToProps: val => ({ content: val }),
   })
 
+  common.implementsTextAlignProp(CardContent, ['left', 'center', 'right'])
```

Comment:
Lets use `_.without(SUI.TEXT_ALIGNMENTS, 'justified')` in tests.

Reply:
Maybe a new constant equals to `['left', 'center', 'right']` is needed?
It is smaller when bundling and faster.

Reply:
I don't think that there is real necessity. This code doesn't work in production builds.